app/src/test/kotlin/com/wire/android/TestUtil.kt

```diff
@@ -19,7 +19,12 @@ package com.wire.android
 
 import kotlin.random.Random
 
-val charPool: List<Char> = ('a'..'z') + ('A'..'Z') + ('0'..'9')
+val charPoolWithNumbers: List<Char> = ('a'..'z') + ('A'..'Z') + ('0'..'9')
+val charPool: List<Char> = ('a'..'z') + ('A'..'Z')
+
+fun Random.stringWithNumbers(length: Int) = (1..length)
+    .map { Random.nextInt(0, charPoolWithNumbers.size).let { charPoolWithNumbers[it] } }
+    .joinToString("")
```

Comment:
issue: Flakyness

This leads to the possibility of schemas starting with numbers and sometimes not starting with numbers, which can become flaky.

Tests will fail at random.

I'd suggest _not_ using random inputs in tests at all, unless we want to get some property testing going on to find the issues.